eng/test-configuration.json

```
@@ -17,6 +17,7 @@
     {"testName": {"contains": "HttpsConnectionClosedWhenResponseDoesNotSatisfyMinimumDataRate"}},
     {"testName": {"contains": "BidirectionalStream_ServerReadsDataAndCompletes_GracefullyClosed"}},
     {"testName": {"contains": "POST_ServerAbort_ClientReceivesAbort"}},
+    {"testName": {"contains": "VariableMultipleStreamsInSequence"}},
```

adding the test that failed in your first go-round @BrennanConroy but it's your choice
```suggestion
    {"testName": {"contains": "VariableMultipleStreamsInSequence"}},
    {"testNAme": {"comtains": "POST_ServerAbortAfterWrite_ClientReceivesAbort"}},
```